shared/api/cluster.go

```diff
@@ -239,3 +245,55 @@ type ClusterMemberStatePost struct {
 	// Example: evacuate
 	Action string `json:"action" yaml:"action"`
 }
+
+// ClusterGroupsPost represents the fields available for a new cluster group.
+//
+// swagger:model
+//
+// API extension: clustering_groups
+type ClusterGroupsPost struct {
```

Comment:
This should include `ClusterGroupPut` so we can set the `Description` as well as `Members` at creation time too.